apps/web/src/components/InternInfo/InternInfo.tsx

```diff
@@ -26,25 +25,38 @@ const InternInfo = ({ intern }: InternInfoProps) => {
           }
           alt="Intern image"
         />
-        <div>
-          <div className={styles.atribute}>
-            <h3>Email</h3>
-            <span> {intern.email} </span>
-          </div>
-          <div>
-            {Object.keys(intern.data).map((key: string) => {
-              return (
-                <div className={styles.atribute}>
-                  <h3>{key}</h3>
-                  <span>
-                    {intern.data[key as keyof typeof intern.data].toString()}
-                  </span>
-                </div>
-              );
-            })}
-          </div>
+        <div className={styles.data}>
+          {Object.keys(intern.data).map((key: string) => {
+            return (
+              <div className={styles.atribute} key={key}>
+                <span>
+                  {key}:{' '}
+                  {intern.data[key as keyof typeof intern.data].toString()}
+                </span>
+              </div>
+            );
+          })}
         </div>
       </div>
+
+      <div className={styles.atribute}>
+        <span>Score: {intern.interviewSlot?.score}</span>
+      </div>
+      {JSON.parse(JSON.stringify(intern.interviewSlot?.answers)).map(
```

Comment:
problem s ovim je što pukne kad intern nije prošao intervju, a ovaj JSON.parse i JSON.stringify riješi TS errore iz razloga što pretvori tip u `any` (jer parse ne zna koji json parsira) pa zato prođe ali iz istog razloga nekad page pukne ako podatak nije array.
jednostavno rjesenje za ovo je samo provjeriti jel `answers` array:
```
          {Array.isArray(intern.interviewSlot?.answers) &&
            intern.interviewSlot?.answers.map((item: Answer) => {
```
tako ce ovaj dio ostat prazan ako intern nije jos imao intervju